shipping/src/main/java/works/weave/socks/ShippingController.java

```diff
@@ -27,7 +27,11 @@ public String getShippingById(@PathVariable String id) {
     @ResponseBody
     Shipment postShipping(@RequestBody Shipment shipment) {
         System.out.println("Adding shipment to queue...");
-        rabbitTemplate.convertAndSend("shipping-task", shipment);
+        try {
+            rabbitTemplate.convertAndSend("shipping-task", shipment);
+        } catch (Exception e) {
```

Comment:
Ok, ignoring. But the exception needs to be caught and ignored, or else the front-end will crash.